lib/usd/utils/ForwardDeclares.h

```diff
@@ -30,6 +30,8 @@ class UsdAttribute;
 class UsdPrim;
 class UsdGeomCamera;
 class UsdLuxDistantLight;
+class UsdPrimCompositionQuery;
+class UsdPrimCompositionQueryArc;
```

Comment:
Forward declare UsdPrimCompositionQuery. UsdPrimCompositionQueryArc classes.

They are useful for interrogating of prims for composition arcs.